packages/core/src/utils/repeat.ts

```diff
@@ -0,0 +1,15 @@
+// LOG-79: Add this into `essentials` package
+// Disable FP rules here to use the performant approach while keep the function itself "FP"
+/* eslint-disable @silverhand/fp/no-let, @silverhand/fp/no-mutation */
+const repeat = <T>(times: number, initial: T, iterate: (accumulator: T) => T) => {
```

Comment:
nit: feels like `new Array(times).fill().reduce(accumulator, initial);`

Reply:
yeah, that's the first thing jumped out but think it would be much more costly to create an array with large length and reduce over it compare to simple `while` loop